apps/studio/src/features/dashboard/SiteList.tsx

```diff
@@ -115,13 +116,47 @@ const SuspendableSiteList = (): JSX.Element => {
       <Flex flexDirection="column" gap="2rem">
         <SimpleGrid columns={3} gap="2.5rem" width="100%">
           {sites.map((site) => (
-            <Card key={site.id} width="100%">
-              <CardHeader>
-                <Link href={`/sites/${site.id}`} as={NextLink}>
-                  {site.name}
-                </Link>
-              </CardHeader>
-            </Card>
+            <Link
+              href={`/sites/${site.id}`}
+              as={NextLink}
+              textDecoration="none"
+              color="links.neutral-hover"
+              _hover={{
+                textDecoration: "none",
+                color: "links.neutral-hover",
+              }}
+            >
+              <Flex
+                key={site.id}
+                flexDirection="column"
+                gap="1rem"
+                width="100%"
+              >
+                <Image
+                  src={site.config.logoUrl}
+                  alt={site.name}
+                  borderRadius="0.5rem"
+                  border="1.5px solid #EDEDED"
```

Comment:
should we use colour tokens

Reply:
do i import from `/apps/studio/src/theme/generated/colours.ts`? or is there a better way to do it?

Reply:
ah my bad, lacking context from my end: 
```ts
border = "1.5px solid"
borderColor = "site.background.base" // just as an example
```